Notebooks/Metrics.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>As per slack, I would like to see the golden metric as false for less than threshold and true for above so that we are trying to minimize the number of accidents served in time.</p><p>A social reasoning is to say that "we aim for zero deaths" rather than saying we aim to get "number of survival to equal number of crashes"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AndOleAnd/Capstone_N_A_P/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>